framework/src/components/storage/defaults/config.js

```diff
@@ -61,7 +61,7 @@ const defaultConfig = {
 	default: {
 		host: 'localhost',
 		port: 5432,
-		database: '',
+		database: 'lisk_dev',
```

Comment:
If you set this default then you can remove this value from tenet. 

https://github.com/LiskHQ/lisk-sdk/blob/4e65ecbd6a5752750ee19f36126b025f84d088be/lisk/config/devnet/config.json#L14